app/managers.py

```diff
@@ -0,0 +1,52 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect(
+            "C:/Users/taras/projects/py-actor-manager/cinema.sqlite"

```

Comment:
You should not use your own path
Correct to `cinema.sqlite`

Reply:
when I use the path cinema.sqlite, I see an error (self._connection.execute(
sqlite3.OperationalError: no such table: actors), how can I fix this?